### PR TITLE
Sphinx Docs

### PR DESCRIPTION
**Pull request by Zyzle**
This ports the existing documentation into sphinx and should go some way to resolving #676

### CODE REVIEW

**Comment by prikhi** [comment]
Beautiful, thanks :)

I built it on rtfd.org, check it out: https://evolus-pencil.readthedocs.org/en/sphinx-docs/

Stay tuned for comments.

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Pencil'
+copyright = u'2015, Colin McCulloch'
```
Comment (prikhi) [inline docs/source/conf.py]: How do you feel about `2015, Pencil Contributers` instead?

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '2.1'
```
Comment (prikhi) [inline docs/source/conf.py]: 2.0 for now

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+# The full version, including alpha/beta/rc tags.
+release = '2.1.0'
```
Comment (prikhi) [inline docs/source/conf.py]: & 2.0.8

File: docs/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+# Additional stuff for the LaTeX preamble.
+#'preamble': '',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+  ('index', 'Pencil.tex', u'Pencil Documentation',
+   u'Colin McCulloch', 'manual'),
```
Comment (prikhi) [inline docs/source/conf.py]: `Pencil Contributers`?

(I'm thinking of changing all authors to `Pencil Contributers` & only listing myself as the maintainer)

File: docs/source/conf.py [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+  ('index', 'Pencil', u'Pencil Documentation',
+   u'Colin McCulloch', 'Pencil', 'One line description of project.',
```
Comment (prikhi) [inline docs/source/conf.py]: Here too, and add a one line description. You can just use the tagline from github if you want: `Multiplatform GUI Prototyping/Wireframing`

**Comment by Zyzle** [comment]
Yeah the © change is fine. Wasn't sure what to set it to so left the sphinx defaults, same goes for the version numbrr

**Comment by Zyzle** [comment]
Latest commit should give those changes. I'd recommend taking the 'evolus' name off the project since we're no longer (I assume) associated with the original developers

**Comment by prikhi** [comment]
Yup, `pencil` is taken so I changed it to `pencil-prototyping`, rtd is just taking its time with it.

Seems like the only other comments I have are small knitpicks that I can just edit as I go through the diff. Could you rebase all this into a single commit? You should be able to do:

``` bash
git remote add upstream git@github.com:prikhi/pencil.git
git fetch upstream develop
git rebase -i upstream/develop
```

Then mark as commits except your first one as `squash` & edit the commit message. Lemme know if you need more info.

**Comment by Zyzle** [comment]
OK squashed down to single commit

**Comment by prikhi** [comment]
Thanks :)

**Pull request merged**